doc/templates/header.html

```diff
@@ -1,12 +1,12 @@
 <html>
 <head>
 <title>
- Security Enhanced Linux Reference Policy
+ Security Enhanced Linux fedora-selinux Policy
```

Comment:
"Security Enhanced Linux Policy for Fedora"
or maybe 
"Fedora based Security Enhanced Linux Policy"